package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "test": "mocha --timeout 150000",
-    "lint": "eslint lib/**.js examples/**.js index.js --fix"
+    "lint": "eslint lib/**/*.js lib/*.js examples/*.js index.js --fix"
```

Comment:
I would rather suggest *excluding* the files you wish to ignore than manually specifying each directory/file.
Also when you provide a directory (e.g. `lib/`) to ESLint, it automatically looks into all the subsidiary folders except for the ones ignored; So there's no need for using specifying `lib/` folder twice.